catroid/src/main/java/org/catrobat/catroid/common/NfcTagData.java

```diff
@@ -29,7 +29,7 @@
 
 	private String name;
 	private String uid;
-	public transient boolean isScanning;
+	private String message;
```

Comment:
The message isn't used yet, right? If so, I suggest to remove it from this PR.
When you plan to implement it, also update methods like `clone`, `toString`, `compareTo` if needed.